api/server.test.js

```diff
@@ -0,0 +1,158 @@
+const db = require(`../data/db-config`)
+const request = require(`supertest`)
+const server = require(`./server`)
+
+const collie = {
+    breed_id: 1,
+    breed: `collie`,
+    size: `large`
+  }
+  const pom = {
+    breed_id: 2,
+    breed: `pomeranian`,
+    size: `small`
+  }
+const aussie = {
+    breed_id: 3,
+    breed: `mini australian shepherd`,
+    size: `medium`
+  }
+const newBreed = {
+  breed: `great dane`, 
+  size: `large`
+}
+const missingSize = {breed: `labrador retriever`}
+const missingBreed = {size: `medium`}
+
+beforeAll(async () => {
+    await db.migrate.rollback()
+    await db.migrate.latest()
+})
+
+beforeEach(async () => {
+    await db.seed.run()
+})
+
+test(`Server working`, async () => {
+    const res = await request(server).get(`/`)
```

Comment:
Nit: Since you are reusing the variable res in multiple tests you can declare it as a let (near the other variables around line 26). That way you don't have to keep redeclaring it.